Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/recent-submissions/RecentSubmissions.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
+import isNil from 'lodash/isNil';
+
+import { IDictionary } from '../../../common/common-types';
+import { IRecentSubmissionsReduxState } from '../../../common/types';
+import { IGetSubmissionsUrlParams } from '../../../common/url-types';
+import { IAuthorizationReduxState } from '../../../redux/features/authorizationSlice';
+import { setCurrentPage } from '../../../redux/features/submissionDetailsSlice';
+import { setSubmissions } from '../../../redux/features/submissionsSlice';
+import {
+    useGetLatestSubmissionsInRoleQuery,
+    useGetLatestSubmissionsQuery, useGetTotalCountQuery,
+} from '../../../redux/services/submissionsService';
+import Heading, { HeadingType } from '../../guidelines/headings/Heading';
+import SubmissionsGrid from '../submissions-grid/SubmissionsGrid';
+
+import SubmissionStateLink from './SubmissionStateLink';
+
+const selectedSubmissionsStateMapping = {
+    1: 'All',
+    2: 'In Queue',
+    3: 'Pending',
+} as IDictionary<string>;
+
+const defaultState = { state: { selectedActive: 1 } };
+
+const RecentSubmissions = () => {
+    const [ selectedActive, setSelectedActive ] = useState<number>(defaultState.state.selectedActive);
```

Comment:
do we need here the default state? wouldn't it be the same if we just pass 1 to the useState initial value?